ci/rpm/test_daos_node.sh

```diff
@@ -15,38 +14,43 @@ elif [ "$(lsb_release -si)" = "openSUSE" ]; then
 fi
 
 set -uex
-sudo $YUM -y install daos-client-"${DAOS_PKG_VERSION}"
+if ! sudo dnf -y install daos-client-"${DAOS_PKG_VERSION}"; then
+    echo "Failed to install daos-client"
+    dnf -y repolist || true
+    dnf -y repoquery --qf %{name}-%{evr}\ %repoid daos-client || true
```

Comment:
(lint) This { is literal. Check expression (missing ;/\n?) or quote it. [SC1083]
(lint) This } is literal. Check expression (missing ;/\n?) or quote it. [SC1083]
(lint) This { is literal. Check expression (missing ;/\n?) or quote it. [SC1083]
(lint) This } is literal. Check expression (missing ;/\n?) or quote it. [SC1083]